src/force-app/main/default/classes/Constants.cls

```diff
@@ -5,6 +5,9 @@ public with sharing class Constants {
     public static final String CASE_STATUS_NEW = 'New';
     public static final String CASE_STATUS_CLOSED = 'Closed';
     public static final String ERROR_PLEASE_CLAIM_CASE_BEFORE_CHANGING_ITS_STATUS = 'Please claim case before changing its status!';
+    public static final String ERROR_INACTIVE_ACCOUNT_UPDATE = 'Account should not be updated if it is inactive!';
+    public static final String YES = 'Yes';
+    public static final String NO = 'No';
```

Comment:
Constant names are too generic. I would recommend renaming to something like ACCOUNT_ACTIVE_YES & ACCOUNT_ACTIVE_NO.